src/EVault/modules/Initialize.sol

```diff
@@ -47,6 +47,14 @@ abstract contract InitializeModule is IInitialize, Base, BorrowUtils {
         vaultStorage.interestFee = DEFAULT_INTEREST_FEE.toConfigAmount();
```

Comment:
let's use `AddressUtils.checkContract` for the line above too:

`if (address(asset).code.length == 0) revert E_BadAddress();`